runtime/near-vm-runner/src/wasmer_runner.rs

```diff
@@ -12,7 +12,7 @@ use wasmer_runtime::Module;
 fn check_method(module: &Module, method_name: &str) -> Result<(), VMError> {
     let info = module.info();
     use wasmer_runtime_core::module::ExportIndex::Func;
-    if let Some(Func(index)) = info.exports.get(method_name) {
+    if let Some(Func(index)) = info.exports.map.get(method_name) {
```

Comment:
All in all this and reference update are the only changes in nearcore, so those are OK, as long as we refer the right guys.